scripts/migration.sh

```diff
@@ -34,6 +34,12 @@ printf "Generating migration boilerplate for patch level $NEW_LEVEL..."
 
 echo "SET NAMES utf8mb4 COLLATE utf8mb4_bin;\n" > "$FWD_SCHEMA"
 echo "-- TODO: Implement your forward migration here\n" >> "$FWD_SCHEMA"
+
+echo "SELECT value INTO @patchlevel FROM `dbMetadata` WHERE `name` = `schema-patch-level`;"
```

Comment:
A couple of tweaks might be needed on this line:

1. By default, echo will just write its output to the console, which isn't what we want. We need to redirect it to the migration file instead.

   The path to the migration file is available in the `FWD_SCHEMA` environment variable and we can redirect the output to it with the `>` or `>>` operators. `>` would clobber any existing data, so we don't want to use that one here as it's not the first line of the file. Let's append `>> "$FWD_SCHEMA"` to each line that you're adding so that it sends the output to the right place (like the preceding line does).

2. I think the SQL might fail when we run it because `@patchLevel` was not defined. You can change this line so that it defines `@patchLevel` at the same time by using `SELECT @patchLevel := value FROM dbMetadata...` instead.